docs/stylesheets/imagelightbox.css

```diff
@@ -22,15 +22,15 @@
 }
 
 .imagelightbox-loading {
-    background-color: rgb(128, 128, 128);
-    background-color: rgba(0, 0, 0, .5);
+    background-color: #9b9b9b;
+    background-color: rgba(130, 130, 130, 0.8);
```

Comment:
I think since we dont support IE8 we can get rid of the double background-color entries and only use the ones with rgba (IE doesnt support rgba). Not sure anyhow how those entries got in there in the first place :-)

Reply:
Yeah, just need to convince csslint that it's ok :)